root/static/scripts/edit/URLCleanup.js

```diff
@@ -388,7 +408,7 @@ const CLEANUPS = {
     },
   },
   'amazon': {
-    match: [new RegExp('^(https?://)?([^/]+\\.)?(amazon\\.(com|ca|co\\.uk|fr|at|de|it|co\\.jp|jp|cn|es|in|com\\.br|com\\.mx|com\\.au)|amzn\\.com)', 'i')],
+    match: [new RegExp('^(https?://)?([^/]+(?<!music)\.)?(amazon\\.(com|ca|co\\.uk|fr|at|de|it|co\\.jp|jp|cn|es|in|com\\.br|com\\.mx|com\\.au)|amzn\\.com)', 'i')],
```

Comment:
What's the `(?<!` syntax? I get `SyntaxError: invalid regexp group` if I paste this in the console, so it seems suspect, but then I'm not sure how the tests are passing.

Reply:
Oops, it is [negated lookbehind](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#special-negative-lookbehind) _(Added in ES2018, not yet supported in Firefox)_.

Reply:
Any suggestions what to use that does work in FF?

Reply:
Moving it to be a lookahead before [^/]+ seems to to do the trick, but let me know if that seems like a mistake :)